control/mpc_lateral_controller/src/mpc_lateral_controller.cpp

```diff
@@ -42,45 +42,44 @@ void update_param(
 }
 }  // namespace
 
-MpcLateralController::MpcLateralController(rclcpp::Node & node) : node_{&node}
+MpcLateralController::MpcLateralController(rclcpp::Node & node) : node_{node}
```

Comment:
With this change, `node_` is copied from `node`, right?
I think these should be the same variable.

Reply:
Sorry, node_ is a reference as you said.